src/main/java/com/expediagroup/hiveberg/IcebergSerDe.java

```diff
@@ -48,11 +49,22 @@ public void initialize(@Nullable Configuration configuration, Properties serDePr
       throw new UncheckedIOException("Unable to resolve table from configuration: ", e);
     }
     this.schema = table.schema();
+    if(table instanceof SnapshotsTable) {
+      try {
+        this.inspector = new IcebergObjectInspectorGenerator().createObjectInspector(schema);
+      } catch (Exception e) {
+        throw new SerDeException(e);
+      }
+    } else {
+      List<Types.NestedField> columns = new ArrayList<>(schema.columns());
+      columns.add(Types.NestedField.optional(9999, "snapshot_id", Types.LongType.get()));
```

Comment:
I used `snapshot_id` here as that's the column name in the Iceberg `SnapshotsTable` so I thought it would be slightly nicer from a user perspective if the virtual column in the regular tables were called the same thing out of consistency